tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -358,5 +392,8 @@ packages the next time it builds the project.
 Profiling your C# code
 ----------------------
 
-- `Mono log profiler <https://www.mono-project.com/docs/debug+profile/profile/profiler/>`_ is available for Linux and macOS. Due to a Mono change, it does not work on Windows currently.
-- External Mono profiler like `JetBrains dotTrace <https://www.jetbrains.com/profiler/>`_ can be used as described `here <https://github.com/godotengine/godot/pull/34382>`_.
+- `Mono log profiler <https://www.mono-project.com/docs/debug+profile/profile/profiler/>`_
+  is available for Linux and macOS. Due to a Mono change, it does not work on
+  Windows currently.
+- External Mono profiler like `JetBrains dotTrace <https://www.jetbrains.com/profiler/>`_
+  can be used as described `here <https://github.com/godotengine/godot/pull/34382>`_.
```

Comment:
Just want to confirm, this is still relevant for Godot 4 even though it uses .net now?

Reply:
Probably not. Like the entire section about external editors, it'll need to be edited. I'll be able to write instructions for VSCode since that's what I use on a daily basis, but for others I'm almost clueless.

Reply:
Do you want to do that in this PR? Or should we merge this now and save that for a different PR?

Reply:
I think it can be done in a subsequent PR. I'll try to gather info from people using other environments in the meantime.

Reply:
Then this one is good to merge now.